### DISCUSSION TITLE
Configuration for saas

### DISCUSSION THREAD

**Discussion by idzczakp** [pocketbase/pocketbase#1318] [Q&A]
Hello!

I'm struggling with understanding how could I use pocketbase for my small saas.

Currently, it's a MERN project and uses Project ID and key to get all posts, pages, users, etc. passed in API header.
I would like to switch from mongo to pocketbase, however, I'm not sure how to achieve the same effect.

As far as I know, I'm not able to pass any parameter for GET requests (but I may be wrong). Maybe there's another option to use one DB for multiple projects?

**Comment by ganigeorgiev** [comment]
You can pass query parameters or headers just fine, but PocketBase doesn't have "projects" level separation since it was not designed to maintain more than one app/project's dataset in a single db file.
Please keep in mind that PocketBase scales only vertically, and depending on your requirements, may not be suitable to have mulitple apps/projects on a single server.

You can of course have as many `base` and `auth` collections as you want (the users in each auth collection are separate and they have their own auth options and constraints) but you'll need to prefix them, eg: app1_users, app2_users, etc. and use the API rules to restrict each collection (in the filters you can access the authenticated user collection name via the `@request.auth.collectionName`). If you haven't already, you can also check https://pocketbase.io/docs/collections/.

In general, I recommend maintaining separate projects per `pb_data` directory. It will be also a lot easier later when you want to move a single project to another server (also note that a single PocketBase executable can be started multiple times with different ports and different `--dir` option).
